org.natron.Natron.json

```diff
@@ -0,0 +1,841 @@
+            ]
+        },
+        {
+            "name": "ilmbase",
+            "config-opts": [
+                "--disable-static"
+            ],
+            "build-options": {
+                "env": {
+                    "LDFLAGS": "-lpthread"
+                }
+            },
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.savannah.nongnu.org/releases/openexr/ilmbase-2.2.0.tar.gz",
+                    "sha256": "ecf815b60695555c1fbc73679e84c7c9902f4e8faa6e8000d2f905b8b86cedc7"
+                },
+                {
+                    "type": "shell",
+                    "commands": [
+                        "rm config.guess configure"
+                    ]
+                }
+            ],
+            "cleanup": [
+                "/include/OpenEXR",
+                "/lib/libHalf.la",
+                "/lib/libIex*.la",
+                "/lib/libIlm*.la",
+                "/lib/libImath*.la",
+                "/lib/pkgconfig/IlmBase.pc"
+            ]
+        },
+        {
+            "name": "openexr",
+            "config-opts": [
+                "--disable-static"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.savannah.nongnu.org/releases/openexr/openexr-2.2.0.tar.gz",
+                    "sha256": "36a012f6c43213f840ce29a8b182700f6cf6b214bea0d5735594136b44914231"
+                },
+                {
+                    "type": "patch",
+                    "path": "patches/openexr-no-docs.patch"
+                },
+                {
+                    "type": "shell",
+                    "commands": [
+                        "rm config.guess configure"
+                    ]
+                }
+            ],
+            "cleanup": [
+                "/bin",
+                "/include",
+                "/lib/*.la",
+                "/lib/pkgconfig",
+                "/share/aclocal"
+            ]
+        },
+        {
+            "name": "openjpeg",
+            "buildsystem": "cmake-ninja",
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=Release"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/uclouvain/openjpeg/archive/v2.3.0.tar.gz",
+                    "sha256": "3dc787c1bb6023ba846c2a0d9b1f6e179f1cd255172bde9eb75b01f1e6c7d71a"
+                }
+            ],
+            "cleanup": [
+                "/include"
+            ]
+        },
+        {
+            "name": "libraw",
+            "config-opts": [
+                "--disable-static",
+                "--enable-jasper",
+                "--enable-jpeg"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://www.libraw.org/data/LibRaw-0.18.9.tar.gz",
```

Comment:
don't forget the demosaic packs for libraw! many users want this. see https://github.com/NatronGitHub/Natron/blob/RB-2.3/tools/jenkins/include/scripts/build-Linux-sdk.sh#L1691